calico/felix/config.py

```diff
@@ -441,7 +441,7 @@ def _validate_cfg(self, final=True):
             if parameter.value is None:
                 # No value, not even a default
                 raise ConfigException("Missing undefaulted value",
-                                      self.parameters["InterfacePrefix"])
+                                      name)
```

Comment:
It looks to me (and admittedly I only gave this a cursory glance) that `self.parameters["InterfacePrefix"]` will be the same, no matter which parameter has been undefaulted.

Looking a bit closer, I think `parameter` is what I should have passed in here. I think I got confused because `name` isn't being used in the loop.